src/AppBundle/Controller/ProcurationManagerController.php

```diff
@@ -44,10 +51,13 @@ public function indexAction(): Response
      * )
      * @Method("GET")
      */
-    public function requestsListAction($page): Response
+    public function requestsListAction(Request $request, $page): Response
     {
-        $requestsRepository = $this->getDoctrine()->getRepository(ProcurationRequest::class);
-        $requests = $requestsRepository->findManagedBy($this->getUser(), (int) $page, self::PER_PAGE);
+        try {
+            $requests = $this->get('app.procuration.manager')->getProcurationRequests(strtolower($request->query->get('type', 'unprocessed')), self::PER_PAGE, (int) $page);
```

Comment:
The int casting should be a parameter type hint in the signature (it will also cast).